octoprint_filamentsensorsimplified/templates/filamentsensorsimplified_navbar.jinja2

```diff
@@ -0,0 +1 @@
+<a title="Waiting..."><i class="muted fas fa-circle"></i></a>
```

Comment:
I think circle is too general, was having a look at a substitute and think that life ring icon that resembles filament spool or tape icon rotated 90 degrees would be better:
<i class="fas fa-life-ring"></i>
or
<i class="fas fa-tape fa-rotate-90"></i>

Reply:
Yes - there is two different styles when filament is out and when ok - so maybe:
When runout:
<span class="fa-stack fa-1x">
  <i class="fas fa-life-ring fa-stack-1x"></i>
  <i class="fas fa-ban fa-stack-2x text-warning"></i>
</span>

When not runout:
<i class="fas fa-life-ring"></i>

Reply:
Take a look - I just committed all the changes

Reply:
And yes the colors is maybe simpler